Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -617,12 +617,11 @@ TextureCacheBase::TCacheEntryBase* TextureCacheBase::Load(const u32 stage)
   // For efb copies, the entry created in CopyRenderTargetToTexture always has to be used, or else
   // it was
   // done in vain.
```

Comment:
A little out of scope, but can you fix that comment anyways so I can sleep better? ;)